packit/api.py

```diff
@@ -1096,6 +1096,9 @@ def _handle_sources(
 
         archives_to_upload = False
         sources_file = self.dg.local_project.working_dir / "sources"
+
+        # Here, dist-git specfile has already been updated.
+        # (=Updates to the sources done in upstream has been synced to dist-git specfile.)
```

Comment:
I don't understand what are "Updates to the sources done in upstream". Updates of "sources" file? But this usually does not exist upstream. Or updates of the `Source:` tag in the spec file? I thought that propose-downstream does not touch this line.

Reply:
> It does. 

Sorry, I misunderstood what you wrote. I thought you meant that Packit updated the `Source:` tag in the upstream spec file. But actually, you probably mean that the current state of that line upstream gets copied downstream without any further alteration (which is what I meant by "propose-downstream does not touch this line").

Reply:
I've reworded the docstrings. PTAL